remotes/docker/schema1/converter.go

```diff
@@ -177,6 +177,7 @@ func (c *Converter) fetchManifest(ctx context.Context, desc ocispec.Descriptor)
 		return err
 	}
 
+	// TODO(cpuguy83): limited reader
 	b, err := ioutil.ReadAll(rc)
```

Comment:
Seems like this should be limited, but I'm not sure what would be reasonable.

Reply:
The limit should be the size in the descriptor, however I am not sure in the v1 case we always have the correct size. If the size is 0 we should probably limit at something high like 8MB

Reply:
8MB would give around 64KB per layer against our 125 layer max, which is more than enough

Reply:
4-8MB is good here.